### ISSUE TITLE
关于attention mask

### ISSUE THREAD

**Issue by gmftbyGMFTBY** [yangjianxin1/GPT2-chitchat#38]
谢谢你的工作
我发现你的代码中没有关于attention_mask的内容，在训练和推理阶段的时候，attention mask应该是必要的，我想问一下，如何加上了会使得效果变好么

**Comment by yangjianxin1** [comment]
该项目的model使用transformers实现，没有从头实现，想了解模型原理的话，可以阅读transformers的项目源码
